src/app/modules/entry-live/views/live-discovery-chart/discovery-chart/discovery-chart.component.ts

```diff
@@ -228,57 +238,64 @@ export class DiscoveryChartComponent {
-          dataBackground: {
-            areaStyle: {
-              color: '#cccccc',
-              opacity: 1,
-            },
-            lineStyle: {
-              color: '#cccccc',
-              opacity: 1,
-            }
-          }
-        }
-      ],
+      ]
     };
   }
-  
+
   private shouldNormalize(metric: string): boolean {
     const normalizedMetrics = ['view_unique_buffering_users', 'view_unique_audience', 'view_unique_audience_dvr', 'view_unique_engaged_users', 'avg_view_live_latency'];
     return normalizedMetrics.indexOf(metric) > -1;
   }
-  
+
   public _onChartInit(chartInstance): void {
     this._echartsIntance = chartInstance;
-    chartInstance.on('datazoom', ({ start, end }) => {
-      this._dataZoomStart = start;
-      this._dataZoomEnd = end;
+
+    // preselect zoom feature in toolbox to enable zooming without clicking the zoom icon
+    setTimeout(() => {
+      this._echartsIntance.dispatchAction({
+        type: 'takeGlobalCursor',
+        key: 'dataZoomSelect',
+        dataZoomSelectActive: true
+      });
+    }, 0);
+
+    chartInstance.on('datazoom', (event) => {
```

Comment:
we probably should unsubscribe from the event upon component destroy to prevent memory leaking
there's `off ` function https://echarts.apache.org/en/api.html#echartsInstance.off in the echart instance